src/Microsoft.Azure.ServiceBus/SessionPumpHost.cs

```diff
@@ -13,13 +13,17 @@ internal sealed class SessionPumpHost
         readonly object syncLock;
         SessionReceivePump sessionReceivePump;
         CancellationTokenSource sessionPumpCancellationTokenSource;
+        readonly string namespaceName;
+        readonly string entityPath;
```

Comment:
>entityPath [](start = 24, length = 10)

potentially add this to the IMessageSessionEntity interface like we do in the IReceiverClient.